rasa/shared/utils/validation.py

```diff
@@ -43,17 +43,16 @@ def __init__(
             content: yaml content loaded from the file (used for line information)
         """
         self.message = message
-        self.filename = filename
         self.validation_errors = validation_errors
         self.content = content
-        super(YamlValidationException, self).__init__()
+        super(YamlValidationException, self).__init__(filename)
```

Comment:
optional: if possible, call the `super()` before the other instructions in the constructor